### PR TITLE
User profile css

### PR DESCRIPTION
**Pull request by charlesLoiseau**

### CODE REVIEW

File: rcongui/src/components/ChatWidget/ChatWidget.js [hunk 1]
```diff
@@ -1,35 +1,26 @@
...
 import Typography from "@material-ui/core/Typography";
 import moment from "moment";
 
 const useStyles = makeStyles((theme) => ({
   message: {
     whiteSpace: "pre-wrap",
-    textAlign: "left"
+    textAlign: "left",
+    marginTop: '5px',
+    marginBottom: '5px',
```
Comment (MarechJ) [inline rcongui/src/components/ChatWidget/ChatWidget.js:14]: Please us material UI's spacing. The whole framework is about getting the spacing consistant :)

File: rcongui/src/components/ChatWidget/ChatWidget.js [hunk 2]
```diff
@@ -1,35 +1,26 @@
...
   },
   date: {
-    color: theme.palette.disabledTextColor,
+    color: theme.palette.disabledColor,
+  },
+  padding: {
+    padding: theme.spacing(1),
+    overflow: "auto",
+    maxHeight: '68vh'
```
Comment (MarechJ) [inline rcongui/src/components/ChatWidget/ChatWidget.js:22]: Why do you need an fixed height?

File: rcongui/src/components/ChatWidget/ChatWidget.js [hunk 3]
```diff
@@ -50,12 +41,12 @@ const ChatContent = ({ data, handleMessageSend }) => {
...
       alignItems="flex-start"
       direction="column"
-      spacing={1}
       className={classes.padding}
+      wrap="nowrap"
     >
       {data?.map((message, index) => {
         return (
-          <Grid item key={index}>
+          <Grid item key={index} xs={12}>
```
Comment (MarechJ) [inline rcongui/src/components/ChatWidget/ChatWidget.js:49]: You won't be able to use the justify from flex box if you specify a size

File: rcongui/src/components/ChatWidget/ChatWidget.js (hunk 2 above)
Comment (charlesLoiseau) [reply rcongui/src/components/ChatWidget/ChatWidget.js:22]: so you just scroll on the comment and not on the page and the form to send comment is always visible

**Pull request merged**